Build failure: mysql-workbench

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build mysql-workbench

### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

https://hydra.nixos.org/build/305823768/nixlog/1

### Relevant log output

```console
[772/788] Linking CXX executable tools/genwrap/genwrap
FAILED: [code=1] tools/genwrap/genwrap 
: && /nix/store/bcw9f6r9v2fm3kv7d15fcrya0mf34xds-gcc-wrapper-14.3.0/bin/g++ -O3 -DNDEBUG -Wl,--dependency-file=tools/genwrap/CMakeFiles/genwrap.dir/link.d tools/genwrap/CMakeFiles/genwrap.dir/genwrap.cpp.o -o tools/genwrap/genwrap  -Wl,-rpath,/build/mysql-workbench-community-8.0.43-src/build/backend/wbpublic:/build/mysql-workbench-community-8.0.43-src/build/library/mtemplate:/build/mysql-workbench-community-8.0.43-src/build/library/forms:/build/mysql-workbench-community-8.0.43-src/build/ext/scintilla/gtk:/build/mysql-workbench-community-8.0.43-src/build/frontend/linux/linux_utilities:/build/mysql-workbench-community-8.0.43-src/build/library/grt/src:/build/mysql-workbench-community-8.0.43-src/build/library/mysql.canvas/src:/build/mysql-workbench-community-8.0.43-src/build/library/base:/build/mysql-workbench-community-8.0.43-src/build/library/cdbc:/build/mysql-workbench-community-8.0.43-src/build/library/parsers  backend/wbpublic/libwbpublic.so.8.0.43  library/mtemplate/libmtemplate.so.8.0.43  /nix/store/xbfyfvj0ws8r2mhw0ci0w26z6lmzpxai-libSM-1.2.6/lib/libSM.so  /nix/store/61fim0684ydsn8g2112xw6mq8sxq675g-libICE-1.1.2/lib/libICE.so  /nix/store/r59pxk0zds4ap8frp8af6f91aykzkhay-libx11-1.8.12/lib/libX11.so  /nix/store/3nn91cvljzhwfsskyp82snq98jial6kk-libxext-1.3.6/lib/libXext.so  -L/nix/store/6z3f1y1sgj4zg1d9k32rnj4ir4b51xa0-mysql-8.0.43/lib -lmysqlclient -lzstd -lssl -lcrypto -lresolv -lm  -L/nix/store/pr8w81b5yb3d8ax53vxya61aq47prh4f-vsqlite-0.3.13/lib -lvsqlitepp  library/grt/src/libgrt.so.8.0.43  /nix/store/qm7kf0nq8d92j9y32gw3v5k38r3fx5xl-libxml2-2.14.5/lib/libxml2.so  -luuid  library/base/libwbbase.so.8.0.43  -L/nix/store/c32blpx8wfra08aqldb3p288xd5f26wx-antlr-runtime-cpp-4.13.2/lib -lantlr4-runtime  -Wl,-rpath-link,/build/mysql-workbench-community-8.0.43-src/build/library/forms:/build/mysql-workbench-community-8.0.43-src/build/ext/scintilla/gtk:/build/mysql-workbench-community-8.0.43-src/build/frontend/linux/linux_utilities:/build/mysql-workbench-community-8.0.43-src/build/library/mysql.canvas/src:/build/mysql-workbench-community-8.0.43-src/build/library/cdbc:/build/mysql-workbench-community-8.0.43-src/build/library/parsers && :
/nix/store/xrwdb41dqi2ia6lr2s61w5bzfg2m71pi-binutils-2.44/bin/ld: cannot find -lzstd: No such file or directory
collect2: error: ld returned 1 exit status
[773/788] Building CXX object plugins/wb.printing/linux/CMakeFiles/wb.printing.wbp.dir/wb_printing_linux.cpp.o
In file included from /build/mysql-workbench-community-8.0.43-src/backend/wbpublic/grt/grt_manager.h:27,
                 from /build/mysql-workbench-community-8.0.43-src/backend/wbpublic/grtui/gui_plugin_base.h:27,
                 from /build/mysql-workbench-community-8.0.43-src/plugins/wb.printing/linux/wb_printing_linux.cpp:26:
/build/mysql-workbench-community-8.0.43-src/library/grt/src/grt.h:1388:40: warning: template-id not allowed for constructor in C++20 [-Wtemplate-id-cdtor]
 1388 |     explicit ListRef<internal::Integer>(const ValueRef &lvalue) : BaseListRef(lvalue) {
      |                                        ^
/build/mysql-workbench-community-8.0.43-src/library/grt/src/grt.h:1388:40: note: remove the '< >'
/build/mysql-workbench-community-8.0.43-src/library/grt/src/grt.h:1816:37: warning: template-id not allowed for constructor in C++20 [-Wtemplate-id-cdtor]
 1816 |     explicit ListRef<internal::Dict>(const ValueRef &lvalue) : BaseListRef(lvalue) {
      |                                     ^
/build/mysql-workbench-community-8.0.43-src/library/grt/src/grt.h:1816:37: note: remove the '< >'
ninja: build stopped: subcommand failed.
```

### Additional context

_No response_

### System metadata

- system: `"x86_64-linux"`
 - host os: `Linux 6.16.1, NixOS, 25.11 (Xantusia), 25.11.20250819.2007595`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.30.2`
 - channels(root): `""`
 - nixpkgs: `/nix/store/pxnx7dbr0pz3qpgw2r315wdbb00afdw7-source`

### Notify maintainers

@TomaSajt 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/my/mysql-workbench/package.nix
+++ b/pkgs/by-name/my/mysql-workbench/package.nix
@@ -6,9 +6,9 @@
   replaceVars,
 
   cmake,
+  jre,
   ninja,
   pkg-config,
-  jre,
   swig,
   wrapGAppsHook3,
 
@@ -17,25 +17,27 @@
   glibc,
   sudo,
 
+  python3Packages,
+
   cairo,
   mysql,
   libiodbc,
   proj,
 
   antlr4_13,
-  gtkmm3,
-  libxml2,
-  libmysqlconnectorcpp,
-  vsqlite,
-  gdal,
   boost,
+  gdal,
+  gtkmm3,
+  libmysqlconnectorcpp,
+  libsecret,
   libssh,
+  libuuid,
+  libxml2,
+  libzip,
   openssl,
   rapidjson,
-  libuuid,
-  libzip,
-  libsecret,
-  python3Packages,
+  vsqlite,
+  zstd,
 }:
 
 let
@@ -94,28 +96,29 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [
     cmake
+    jre
     ninja
     pkg-config
-    jre
     swig_4_2
     wrapGAppsHook3
   ];
 
   buildInputs = [
     antlr4_13.runtime.cpp
-    gtkmm3
-    (libxml2.override { enableHttp = true; })
-    libmysqlconnectorcpp
-    vsqlite
-    gdal
     boost
+    gdal
+    gtkmm3
+    libiodbc
+    libmysqlconnectorcpp
+    libsecret
     libssh
+    libuuid
+    (libxml2.override { enableHttp = true; })
+    libzip
     openssl
     rapidjson
-    libuuid
-    libzip
-    libsecret
-    libiodbc
+    vsqlite
+    zstd
 
     bash # for shebangs
 

